src/main/java/com/auth0/client/mgmt/ManagementAPI.java

```diff
@@ -143,10 +144,35 @@ public void setTelemetry(Telemetry telemetry) {
      *
      * @param enabled whether to enable the HTTP logger or not.
      */
+    @Deprecated
     public void setLoggingEnabled(boolean enabled) {
         logging.setLevel(enabled ? Level.BODY : Level.NONE);
     }
 
+    private void configureLogging(LoggingOptions loggingOptions) {
+        if (loggingOptions == null) {
+            logging.setLevel(Level.NONE);
+            return;
+        }
+        switch (loggingOptions.getLogLevel()) {
+            case BASIC:
+                logging.setLevel(Level.BASIC);
+                break;
+            case HEADERS:
+                logging.setLevel(Level.HEADERS);
+                break;
+            case BODY:
+                logging.setLevel(Level.BODY);
+                break;
+            case NONE:
+            default:
+                logging.setLevel(Level.NONE);
+        }
+        for (String header : loggingOptions.getHeadersToRedact()) {
+            logging.redactHeader(header);
+        }
+    }
+
```

Comment:
Yes, unfortunately this (and all other networking client config) is copy-paste between the Auth and Management API clients :(. Given the current package structure, extracting it to a common ancestor or utility class would require exposing public methods and dependencies (e.g., OkHttp) that we do not wish to do.